tests/utils/main_test.go

```diff
@@ -0,0 +1,22 @@
+//go:build e2e
+// +build e2e
+
+package utils
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	. "github.com/kedacore/keda/v2/tests/helper"
+)
+
+func TestMain(m *testing.M) {
```

Comment:
might be interesting to see if we can move this to either `helper` package or some common test package later